politeiad/config.go

```diff
@@ -441,6 +492,19 @@ func loadConfig() (*config, []string, error) {
 	// duplicate addresses.
 	cfg.Listeners = normalizeAddresses(cfg.Listeners, port)
 
+	// Load configurable mime types to politeiad.
+	if len(cfg.MimeTypes) == 0 {
+		cfg.MimeTypes = defaultMimeTypes
+	} else {
+		mimes, err := parseAndValidateMimeTypes(cfg.MimeTypes)
+		if err != nil {
+			return nil, nil, err
+		} else {
+			cfg.MimeTypes = mimes
```

Comment:
There's no need for this else clause.  It's preferred to keep indentation to the right to a minimum.  It makes it easier to read.  So instead of this:
```
mimes, err := parseAndValidateMimeTypes(cfg.MimeTypes)
if err != nil {
  return nil, nil, err
} else {
  cfg.MimeTypes = mimes
}
```
Do this:
```
mimes, err := parseAndValidateMimeTypes(cfg.MimeTypes)
if err != nil {
  return nil, nil, err
}
cfg.MimeTypes = mimes
```